app/services/users/remove_role.rb

```diff
@@ -0,0 +1,49 @@
+module Users
+  class RemoveRole
+    attr_reader :user, :role, :resource_type, :current_user, :response
```

Comment:
Nitpick: you could move this line below `private`, I don't think these need to be accessible from the outside.